ravenframework/Models/EnsembleModel.py

```diff
@@ -534,10 +539,30 @@ def submit(self,myInput,samplerType,jobHandler,**kwargs):
     ## works, we are unable to pass a member function as a job because the
     ## pp library loses track of what self is, so instead we call it from the
     ## class and pass self in as the first parameter
-    if self.parallelStrategy == 1:
-      jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, kwargs)
-    else:
-      jobHandler.addClientJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, kwargs)
+
+    nRuns = 1
+    batchMode =  kwargs.get("batchMode", False)
+    if batchMode:
+      nRuns = kwargs["batchInfo"]['nRuns']
+
+    for index in range(nRuns):
+      if batchMode:
+        kw =  kwargs['batchInfo']['batchRealizations'][index]
+      else:
+        kw = kwargs
+
+      prefix = kw.get("prefix")
+      uniqueHandler = kw.get("uniqueHandler",'any')
+      forceThreads = kw.get("forceThreads",False)
+
+      metadata = kw
+
+      if self.parallelStrategy == 1:
+        jobHandler.addJob((self, myInput, samplerType, kw), self.__class__.evaluateSample, prefix, metadata=metadata,
```

Comment:
In addition, we need add a test for this PR to test the GA with ensemble model

Reply:
test added